app/jobs/moab_replication_audit_job.rb

```diff
@@ -0,0 +1,42 @@
+# Confirms that a CompleteMoab is fully/properly replicated to all target zip endpoints.
+# Usage info:
+# ReplicatedFileCheck.set(queue: :endpoint_check_us_west_2).perform_later(cm)
+# ReplicatedFileCheck.set(queue: :endpoint_check_us_east_1).perform_later(cm)
```

Comment:
The class name looks wrong to me in this usage info since I see `ReplicatedFileCheck` gets deleted.